plrustc/README.md

```diff
@@ -26,56 +24,26 @@ We have a few independent workspaces. My `rust-analyzer` configuration currently
-
-2. Then, install Rust for that verion via an offline installer from <https://forge.rust-lang.org/infra/other-installation-methods.html#standalone-installers>.
-
-3. Then, install the `rustc-dev` component for the same version from the distribution server. A user-friendly link to this is not available (although the steps to determine the URL are documented [here](https://forge.rust-lang.org/infra/channel-layout.html)), but this generally will be something like `https://static.rust-lang.org/dist/rustc-dev-$RUST_VERSION-$TARGET_TRIPLE.tar.gz`, where `RUST_VERSION` is the version from the other step, and TARGET_TRIPLE is either `x86_64-unknown-linux-gnu` (on x86_64 linux), and `aarch64-unknown-linux-gnu` (on aarch64 linux).
-
-    Note: Other targets are not really supported by PL/Rust trusted mode at the moment (aside from partial checking-only support for development purposes, which lacks documentation).
-
-4. Then, build and install `plrustc` and `postgrestd`. The build system at the root of the repository can help. `./b.sh help`. In theory, it can do a few of the other steps in this list too, but for safety I've written them down anyway.
-
-5. After all that, you can finally install PL/Rust. I think this is done with `cargo pgx install`? Something like that. Someone should clean up this guide honestly.
-
 ## Usage
 
-Similar to `rustc`, `plrustc` is usually not invoked directly, but instead through `cargo`. While a `plrustc`-using wrapper for cargo exists in this repository, it should not be used except for testing/development/etc.
+Similar to `rustc`, `plrustc` is usually not invoked directly, but instead through `cargo`.
 
 ## Details
 
-Some additional details are provided for users who intend to run PL/Rust and plrustc under restricted environments via seccomp and/or selinux. These details are subject to change, which should be noted in documentation.
-
-### Environment variables
-
-Here are some `plrustc`-specific environment variables that PL/Rust or `plrustc` itself may set. This is in addition to the ones mentioned in "Sysroot Configuration" below.
-
-The `PLRUSTC_FLAGS` variable contains flags passed by PL/Rust which are specific to `plrustc` and required for it to function. The `PLRUSTC_PASSTHROUGH` argument is also used to invoke `plrustc` as if it were plain `rustc`, which it may . FIXME(thom): `PLRUSTC_FLAGS` are not used right now.
-
-PL/Rust is likely to invoke `plrustc` as the value of both `RUSTC` and
-`RUSTC_WRAPPER`. FIXME(thom): seems like we might only need to set it as `RUSTC` now? Not sure.
+Some additional details are provided for users who intend to run PL/Rust and plrustc under restricted environments via seccomp and/or selinux. These details are subject to change, although if that occurs it will be noted in the changelog.
```

Comment:
```suggestion
Some additional details are provided for users who intend to run PL/Rust and plrustc under restricted environments via seccomp and/or SELinux. These details are subject to change, although if that occurs it will be noted in the changelog.
```